src/inspector/models/notation/bends/bendsettingsmodel.h

```diff
@@ -26,6 +26,7 @@
 
 #include "types/bendtypes.h"
 
+namespace mu::engraving {
```

Comment:
I don't see any usage of objects from the engraving namespace in the header. 
Also, we don't define the engraving namespace in the headers this way. We add the namespace to objects where we use them.

Reply:
So you recommend me to use forward declaration here?

Reply:
fixed